README.md

```diff
@@ -24,11 +24,8 @@ You may be looking for:
 
 ## Serde in action
 
-<details>
-<summary>
 Click to show Cargo.toml.
```

Comment:
```suggestion
```
I believe this line was used to summarize the `details` button/link thingy. At the moment it just shows up like regular text in front of the playground link, so it should probably go away